pkgs/development/python-modules/cyipopt/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cython,
+  ipopt,
+  numpy,
+  pkg-config,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "cyipopt";
+  version = "1.5.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "mechmotum";
+    repo = "cyipopt";
+    rev = "v${version}";
+    hash = "sha256-ddiSCVzywlCeeVbRJg2wxKIlAVlZw9Js95IbEDqhh5Q=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ ipopt ];
+
+  build-system = [
+    cython
+    numpy
+    setuptools
+  ];
+
```

Comment:
Could you add `numpy` to `dependencies`?
https://github.com/mechmotum/cyipopt/blob/9fe5b202d51294bfcc7b0d20d560eea49b2c344b/setup.py#L56-L58